src/types/misc.ts

```diff
@@ -85,8 +85,8 @@ export interface UnboundedZone {
 }
 
 export interface ZoneDimension {
-  height: HeaderIndex;
-  width: HeaderIndex;
+  nOfRows: HeaderIndex;
```

Comment:
not really fond of that name, guess we could see with the others what they think.. @odoo/rd-spreadsheet

Reply:
changed to numberOfCols/numberOfRows